modules/auth/password/password.go

```diff
@@ -121,15 +122,15 @@ func Generate(n int) (string, error) {
 }
 
 // BuildComplexityError builds the error message when password complexity checks fail
-func BuildComplexityError(locale translation.Locale) string {
+func BuildComplexityError(locale translation.Locale) template.HTML {
```

Comment:
`flashMsgStringOrHTML` will convert the output to a string. Is that expected?

Reply:
Yes, that's the old behavior of the "Flash" package. The "message" will be rendered by `Str2html`, so there is no problem at the moment. See the `alert.tmpl`.